src/native/corehost/fxr/command_line.cpp

```diff
@@ -353,16 +353,16 @@ void command_line::print_muxer_usage(bool is_sdk_present)
     for (const auto& opt : known_opts)
     {
         const host_option &arg = get_host_option(opt);
-        trace::println(_X("  %s %-*s  %s"), arg.option, 29 - (int)pal::strlen(arg.option), arg.argument, arg.description);
+        trace::println(_X("  %s %-*s  %s"), arg.option, 30 - (int)pal::strlen(arg.option), arg.argument, arg.description);
     }
-    trace::println(_X("  --list-runtimes                 Display the installed runtimes"));
-    trace::println(_X("  --list-sdks                     Display the installed SDKs"));
+    trace::println(_X("  --list-runtimes [--arch <arch>]  Display the installed runtimes."));
+    trace::println(_X("  --list-sdks [--arch <arch>]      Display the installed SDKs."));
```

Comment:
```suggestion
    trace::println(_X("  --list-runtimes [--arch <arch>]  Display the installed runtimes matching the host or specified architecture. Valid options are x64 and arm64."));
    trace::println(_X("  --list-sdks [--arch <arch>]      Display the installed SDKs matching the host or specified architecture. Valid options are x64 and arm64."));
```

Would it be helpful to include this information? Or, should we include this elsewhere?

Reply:
I started with listing all valid arch values, but there are a decent number and listing them all didn't seem that helpful: https://github.com/dotnet/runtime/blob/1713d65316467e3eaf23514b6642eba5869f1b70/src/native/corehost/hostmisc/utils.cpp#L194-L205

Maybe just listing common ones as examples?
`Display the installed runtimes matching the host or specified architecture. Example architectures: arm64, x64, x86`

Reply:
I like this suggestion! To me it would be valuable to document the values it accepts somewhere and what string format they are in because different ecosystems can describe the same architecture in different ways. I never would have expected 'rscv64' to be a valid value, though I don't work in anything related to that space. If I was a dev outside of dotnet and wanted to use this, there's a chance I'd end up at this comment thread to resolve that question 😁